core/vtk/ttkAlgorithm/ttkTriangulationFactory.cpp

```diff
@@ -76,7 +76,11 @@ struct ttkOnDeleteCommand : public vtkCommand {
     if(this->observee)
       this->observee->RemoveObserver(this);
 
-    auto instance = ttkTriangulationFactory::Instance;
+    auto instance = &ttkTriangulationFactory::Instance;
```

Comment:
@JonasLukasczyk 
can you please confirm you're ok with these edits to this class?